app/src/main/java/com/kickstarter/ui/activities/ChangeEmailActivity.kt

```diff
@@ -7,62 +7,65 @@ import android.view.MenuItem
 import android.view.View
 import android.view.inputmethod.InputMethodManager
 import androidx.core.content.ContextCompat
+import androidx.core.view.isGone
 import com.kickstarter.R
+import com.kickstarter.databinding.ActivityChangeEmailBinding
 import com.kickstarter.libs.BaseActivity
 import com.kickstarter.libs.qualifiers.RequiresActivityViewModel
-import com.kickstarter.libs.utils.ViewUtils
 import com.kickstarter.ui.extensions.onChange
 import com.kickstarter.ui.extensions.showSnackbar
 import com.kickstarter.viewmodels.ChangeEmailViewModel
-import kotlinx.android.synthetic.main.activity_change_email.*
-import kotlinx.android.synthetic.main.change_email_toolbar.*
 import rx.android.schedulers.AndroidSchedulers
 
 @RequiresActivityViewModel(ChangeEmailViewModel.ViewModel::class)
 class ChangeEmailActivity : BaseActivity<ChangeEmailViewModel.ViewModel>() {
 
     private var saveEnabled = false
 
+    private lateinit var binding: ActivityChangeEmailBinding
```

Comment:
Something off with the ChangeEmailActivity, on the lates master I can request the email change and shows a UI saying a verification email has been sent. But in this branch it tells me the password is incorrect and do not let me change the email.

Reply:
Fixed